Userland/Libraries/LibWeb/Layout/InlineFormattingContext.cpp

```diff
@@ -168,6 +168,8 @@ void InlineFormattingContext::dimension_box_on_line(Box const& box, LayoutMode l
     if (height_value.is_auto()) {
         // FIXME: (10.6.6) If 'height' is 'auto', the height depends on the element's descendants per 10.6.7.
         parent().compute_height(box, AvailableSpace(AvailableSize::make_indefinite(), AvailableSize::make_indefinite()));
+    } else if (should_treat_height_as_auto(box, *m_available_space)) {
+        box_state.set_content_height(independent_formatting_context->automatic_content_height());
```

Comment:
Sorry, what I meant to ask is if we can simply change the `height_value.is_auto()` branch above to use `should_treat_height_as_auto`, instead of adding a new branch.

Reply:
yes, we can!